### PR TITLE
Update simd.dd

### PR DESCRIPTION
**Pull request by jmh530**
Fleshing out the importance of runtime checks with SIMD by means of example.

### CODE REVIEW

File: simd.dd [hunk 1]
```diff
@@ -60,6 +60,9 @@ import core.simd;
 	the compiler is targetting. If a particular CPU family has varying
 	support for vector types, an additional runtime check may be necessary.
 	The compiler does not emit runtime checks; those must be done by the programmer.
+    For instance, the Windows version of DMD only implements support for SIMD when
```
Comment (CyberShadow) [inline simd.dd]: Does the OS really matter?

File: simd.dd [hunk 2]
```diff
@@ -60,6 +60,9 @@ import core.simd;
...
+    compiling with the 64bit flag. If a program does not contain runtime checks,
```
Comment (CyberShadow) [inline simd.dd]: "the 64bit flag" is ambiguous

Comment (CyberShadow) [inline simd.dd]: `core.simd` is wrapped in `version(D_SIMD)`, so it shouldn't even compile without `-m64`

File: simd.dd (hunk 1 above)
Comment (jmh530) [inline simd.dd]: I don't know how true it is on other machines. I was just providing an example for what I was working on.

File: simd.dd (hunk 2 above)
Comment (jmh530) [inline simd.dd]: You're right that the "the 64bit flag" could be replaced with `-m64`. I'm new to github (I literally joined to make this change) and it's not entirely obvious to me how to fix it. 

Nevertheless, the general point remains. That page doesn't mention the importance of compiling with `-m64` anywhere and the error messages aren't particularly clear either. I realized I was making a mistake and thought the documentation would be improved if other people know that SIMD might require adding a compiler flag.

Comment (CyberShadow) [inline simd.dd]: > You're right that the "the 64bit flag" could be replaced with -m64. I'm new to github (I literally joined to make this change) and it's not entirely obvious to me how to fix it.

I'm not sure you can fix it from the website alone, you'll probably need a Git client. [GitHub for Windows](https://windows.github.com/) should be simple enough for new Git users.

> I realized I was making a mistake and thought the documentation would be improved if other people know that SIMD might require adding a compiler flag.

OK, but we shouldn't conflate runtime errors with compiler errors.

Comment (CyberShadow) [inline simd.dd]: Maybe such a change belongs in the documentation for the `core.simd` module instead, or to the documentation of the `D_SIMD` built-in version.

Comment (jmh530) [inline simd.dd]: I had decided to add it there because that documentation is the most
detailed. The other ones are more terse. Nevertheless, I would recommend
adding some kind of information about this to all three of those locations.
I hadn't thought of it until you mentioned it.

I have a Github client on my home computer. I doubt it will work on my work
computer (firewall).

Comment (jmh530) [inline simd.dd]: I'm not sure how to modify this one, but I made some changes and will submit in a separate pull request. I'm closing this one.
